pages/api/avatar.md

```diff
@@ -18,14 +18,14 @@ import Avatar from '@material-ui/core/Avatar';
 
 | Name | Type | Default | Description |
 |:-----|:-----|:--------|:------------|
-| <span class="prop-name">alt</span> | <span class="prop-type">string</span> |   | Used in combination with `src` or `srcSet` to provide an alt attribute for the rendered `img` element. |
```

Comment:
@oliviertassinari That's the old `defaultValue` right? I don't think that adds any value to the markdown. Should be fine with no space at all here.

Reply:
:)